docs/narr/startup.rst

```diff
@@ -132,7 +132,7 @@ Here's a high-level time-ordered overview of what happens when you press
 #. ``pserve`` starts the WSGI *server* defined within the ``[server:main]``
    section.  In our case, this is the Waitress server (``use =
    egg:waitress#main``), and it will listen on all interfaces on port 6543
-   for both IPv4 and IPv6 (``listen = 127.0.0.1:6543 [::1]:6543``). The server
+   for both IPv4 and IPv6 (``listen = localhost:6543``). The server
```

Comment:
@bertjwregeer yes, I had to disable the IPv6 stack completely on my linux systems, because some apps wanted to resolve IPv6 addresses, which the router didn't pass, resulting in nasty stalls. Here's the traceback:
```
(env) hp@host:~/src/python3/web/wiki> pserve development.ini --reload
Starting monitor for PID 6260.
Starting server in PID 6260.
Process Process-1:
Traceback (most recent call last):
  File "/usr/lib64/python3.4/multiprocessing/process.py", line 254, in _bootstrap
    self.run()
  File "/usr/lib64/python3.4/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/home/hp/src/python3/web/wiki/env/lib64/python3.4/site-packages/hupper/worker.py", line 253, in worker_main
    func(*spec_args, **spec_kwargs)
  File "/home/hp/src/python3/web/wiki/env/lib64/python3.4/site-packages/pyramid/scripts/pserve.py", line 40, in main
    return command.run()
  File "/home/hp/src/python3/web/wiki/env/lib64/python3.4/site-packages/pyramid/scripts/pserve.py", line 232, in run
    server(app)
  File "/home/hp/src/python3/web/wiki/env/lib64/python3.4/site-packages/paste/deploy/loadwsgi.py", line 189, in server_wrapper
    **context.local_conf)
  File "/home/hp/src/python3/web/wiki/env/lib64/python3.4/site-packages/paste/deploy/util.py", line 55, in fix_call
    val = callable(*args, **kw)
  File "/home/hp/src/python3/web/wiki/env/lib64/python3.4/site-packages/waitress/__init__.py", line 20, in serve_paste
    serve(app, **kw)
  File "/home/hp/src/python3/web/wiki/env/lib64/python3.4/site-packages/waitress/__init__.py", line 11, in serve
    server = _server(app, **kw)
  File "/home/hp/src/python3/web/wiki/env/lib64/python3.4/site-packages/waitress/server.py", line 85, in create_server
    sockinfo=sockinfo)
  File "/home/hp/src/python3/web/wiki/env/lib64/python3.4/site-packages/waitress/server.py", line 182, in __init__
    self.bind_server_socket()
  File "/home/hp/src/python3/web/wiki/env/lib64/python3.4/site-packages/waitress/server.py", line 294, in bind_server_socket
    self.bind(sockaddr)
  File "/usr/lib64/python3.4/asyncore.py", line 334, in bind
    return self.socket.bind(addr)
OSError: [Errno 99] Cannot assign requested address
Server with PID 6260 exited with code 1.
Waiting for changes before reloading.
```